File: C/PRAK404-22101817310001-MuhammadRyanRizkyRahmadi.c
#include <stdio.h>
int main (void){
    float r, y, a;
    while(1){
    printf("Pilih Program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");
    printf("Masukkan Pilihan =\t");
    scanf("%f", &r);
    if(r > 5){
        printf("Input anda salah, silahkan coba lagi\n");
        continue;
    }else if (r == 5){
        printf("Terima kasih, telah menggunakan kalkulator Muhammad Ryan Rizky Rahmadi\n");
        break;
    }
    printf("Masukkan Nilai Pertama =");
    scanf("%f", &y);
    printf("Masukkan Nilai Kedua =");
    scanf("%f", &a);
    if(r == 1){
        printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n",y,a, y+a);
    }else if(r==2){
        printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n",y,a, y-a);
    }else if(r==3){
        printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n",y,a, y*a);
    }else if (r==4){
        printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n",y,a, y/a);

    }
    

    }
}

File: Python/PRAK404-22101817310001-MuhammadRyanRizkyRahmadi.py
while True:
    print("Pilih Program\n 1.Penjumlahan\n 2.Pengurangan\n 3.Perkalian\n 4.Pembagian\n 5.Exit")
    r=int(input("Masukkan Pilihan ="))
    if r > 5:
        print("Input anda salah, silahkan coba lagi")
        continue
    elif r==5:
        print("Terima kasih, telah menggunakan kalkulator Muhammad Ryan Rizky Rahmadi")
        break
    else:
        y=float(input("Masukkan Nilai Pertama ="))
        a=float(input("Masukkan Nilai Kedua   ="))
        if r==1:
            print("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f"% (y,a, y+a))
        elif r==2:
            print("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f"% (y,a, y-a))
        elif r==3:
            print("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f"% (y,a, y*a))
        elif r==4:
            print("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f"% (y,a, y/a))
